content/applications/finance/accounting/bank/feeds/bank_synchronization.rst

```diff
@@ -148,14 +143,15 @@ The synchronization is not working in real-time. Is that normal?
 
 The process is not intended to work in real-time as third party providers synchronize your accounts
 at different intervals. To force the synchronization and fetch the statements, go to your
-*Accounting dashboard*, and click on the *Synchronize Now* button. You can also synchronize and
-fetch transactions through :menuselection:`Accounting --> Configuration --> Online Synchronization`.
-Some providers only allow one refresh per day, so it is possible that clicking on *Synchronize Now*
-does not get your latest transactions if you already performed such action earlier in the day.
+Accounting dashboard, and click on the :guilabel:`Synchronize Now` button. You can also synchronize
+and fetch transactions through :menuselection:`Configuration --> Online Synchronization`.
+Some providers only allow one refresh per day, so it is possible that clicking on
+:guilabel:`Synchronize Now` does not get your latest transactions if you already performed such
+action earlier in the day.
 
 A transaction can be visible on your bank account, but not be fetched if it has the status
-*Pending*. Only transactions with the *Posted* status will be retrieved. If it is not *Posted* yet,
-you will have to wait until the status changes.
+**Pending**. Only transactions with the **Posted** status will be retrieved. If it is not **Posted**
+yet, you will have to wait until the status changes.
```

Comment:
Wouldn't those be guilabel? (I think the status appear on the app. Not sure about the last posted at all so I left it as ** **
```suggestion
A transaction can be visible on your bank account but not be fetched if it has the status
:guilabel:`Pending`. Only transactions with the :guilabel:`Posted` status will be retrieved. If the
transaction is not **Posted** yet, you will have to wait until the status changes.
```